Dockerfile

```diff
@@ -13,26 +13,20 @@ RUN yum update -y && yum install -y \
 # Permit access to fsl installation script (must exist locally)
 ADD ./myFslInstallerScript.sh /myFslInstallerScript.sh
 
-# Fsl-installer needs access to shell variable
+
 ENV SHELL /bin/bash
 
 # Install fsl to default dir
 RUN echo -e "/usr/local/src" | ./myFslInstallerScript.sh
 
 #Clone HCP pipelines
-RUN git clone https://github.com/Washington-University/Pipelines.git pipelines 
-RUN cp -r pipelines /usr/local/src/ 
-RUN rm -rf pipelines
+RUN git clone https://github.com/Washington-University/Pipelines.git pipelines;cp -r pipelines /usr/local/src/;rm -rf pipelines;
```

Comment:
While keeping '\' this error gets thrown.
The command '/bin/sh -c git clone https://github.com/Washington-University/Pipelines.git pipelines     cp -r pipelines /usr/local/src/     rm -rf pipelines' returned a non-zero code: 129